crates/sui-protocol-config/src/lib.rs

```diff
@@ -2820,6 +2821,13 @@ impl ProtocolConfig {
                 62 => {
                     cfg.feature_flags.relocate_event_module = true;
                 }
+                63 => {
+                    cfg.feature_flags.relocate_event_module = true;
```

Comment:
Fix？

Reply:
cc @arun-koshy

Reply:
Missed this, thanks for pointing it out. @halfprice is going to fix it when he moves this feature flag to v64